api/gloo-fed/fed.enterprise.gloo/v1/auth_config.proto

```diff
@@ -10,12 +10,12 @@ option (extproto.equal_all) = true;
 
 import "github.com/solo-io/solo-apis/api/skv2-enterprise/v1alpha1/multicluster.proto";
 import "github.com/solo-io/solo-apis/api/gloo-fed/fed/core/v1/placement.proto";
-import "github.com/solo-io/solo-apis/api/gloo/enterprise.gloo/v1/auth_config.proto";
+import "github.com/solo-io/solo-apis/api/gloo/enterprise.gloo.apis/v1/auth_config.proto";
```

Comment:
As far as I can tell, we're not syncing gloo-fed apis so I made changes in this directory manually.